packages/appeals-service-api/src/routes/v2/users/_id/appeal-submissions/_appealSubmissionId/repo.js

```diff
@@ -0,0 +1,55 @@
+const { createPrismaClient } = require('#db-client');
+const { PrismaClientKnownRequestError } = require('@prisma/client/runtime/library');
+
+/**
+ * @typedef {import("@prisma/client").Prisma.AppealUserGetPayload<{
+ *   include: {
+ *     Appeals: {
+ *       include: {
+ *         Appeal: true
+ *       }
+ *     }
+ *   }
+ * }>} UserWithAppeals
```

Comment:
👍

Reply:
Changed this due to Bryn's suggestion but we still have the type